csharp/src/Ice/Communicator-LocatorManager.cs

```diff
@@ -12,437 +13,366 @@ namespace ZeroC.Ice
 {
     public sealed class LocatorInfo
```

Comment:
Does it need to be public or can we switch to internal?

Reply:
It should be internal, fixed.